src/borg/archiver.py

```diff
@@ -3525,8 +3530,9 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
         .. note::
 
             If you copy a repository with the lock held, the lock will be present in
-            the copy, obviously. Thus, before using borg on the copy, you need to
-            use "borg break-lock" on it.
+            the copy. Thus, before using borg on the copy from or on a different host,
```

Comment:
not sure about what "from or on" shall mean here.

if you run the client "from" a machine, it means the same as if you run the client "on" a machine.